winterfell/webapp/src/styles/header.styl

```diff
@@ -108,13 +108,7 @@
       cursor: pointer
 
 #banner
-  background: repeating-linear-gradient(
-    45deg,
-    colorPrimaryBlue,
-    colorPrimaryBlue 10px,
-    colorPrimaryDarkestBlue 10px,
-    colorPrimaryDarkestBlue 20px
-  );
+  background: colorDarkGray
```

Comment:
sorry I think the gradient is way too sharp and distracting. 😖 Turned it into a more neutral color and the text will still be more viewable.